### PR TITLE
Adding MSTR stake pool

### PR DESCRIPTION
**Pull request by Vardominator**

### CODE REVIEW

File: registry.json [hunk 1]
```diff
@@ -3350,5 +3350,20 @@
...
+    "social": {
+      "facebook": "",
+      "twitter": "MStakepool",
+      "website": "http://www.maestro-pool.com",
+      "reddit": "https://www.reddit.com/user/MAESTRO-stake-pool",
+      "instagram": "",
+      "youtube": ""
+    },
+    "telegram": "MAESTRO_Pool",
+    "operatorName": "MGAV",
```
Comment (Bez625) [inline registry.json:3366]: Please make this your real full name, real first name or leave blank

File: registry.json [hunk 2]
```diff
@@ -3350,5 +3350,20 @@
...
+    "poolId": "abea730f5c32157c76c876597d0b7093d50754d0274d67c62d7434b9",
+    "social": {
+      "facebook": "",
+      "twitter": "MStakepool",
+      "website": "http://www.maestro-pool.com",
+      "reddit": "https://www.reddit.com/user/MAESTRO-stake-pool",
+      "instagram": "",
+      "youtube": ""
+    },
+    "telegram": "MAESTRO_Pool",
```
Comment (Bez625) [inline registry.json:3365]: I couldnt find this username on telegram, please can you make sure it is the handle of an individual and not a group or channel?

**Review by Bez625 — CHANGES_REQUESTED** [review]

File: registry.json (hunk 1 above)
Comment (Vardominator) [reply registry.json]: @Bez625 done!

File: registry.json (hunk 2 above)
Comment (Bez625) [reply registry.json:3365]: @Vardominator Not sure if you saw this comment too? I still can't see MAESTRO_Pool as a user on TG

Comment (Vardominator) [reply registry.json]: @Bez625 was just getting to this. You should be able to find `varderes` now!

**Review by Bez625 — APPROVED** [review]

**Comment by Bez625** [comment]
@Vardominator Unfortunately I cant resolve the conflicts, please can you do so?

**Comment by Vardominator** [comment]
@Bez625 I have resolved the conflict!

**Comment by Vardominator** [comment]
@Bez625 can you please approve? :)

**Pull request merged**